### PR TITLE
Add documentation for all TeamsCallHoldPolicy cmdlets

### PR DESCRIPTION
**Pull request by joelhmarquez**
- Add documentation for all TeamsCallHoldPolicy cmdlets.
- Update TOC in skype.md.

### CODE REVIEW

File: skype/skype-ps/skype/grant-CsTeamsCallHoldPolicy.md
Comment (get-itips) [inline skype/skype-ps/skype/grant-CsTeamsCallHoldPolicy.md:221]: For links to files in the same doc set (folder), use file-relative links (include the file name extension). Example: (New-CsTeamsCallHoldPolicy.md

File: skype/skype-ps/skype/Get-CsTeamsCallHoldPolicy.md
Comment (get-itips) [inline skype/skype-ps/skype/Get-CsTeamsCallHoldPolicy.md:35]:
```suggestion
Assigning a Teams call hold policy to a user sets an audio file to be played during the duration of the hold. 

```

File: skype/skype-ps/skype/New-CsTeamsCallHoldPolicy.md
Comment (get-itips) [inline skype/skype-ps/skype/New-CsTeamsCallHoldPolicy.md:36]:
```suggestion
PS C:\> New-CsTeamsCallHoldPolicy -Identity "ContosoPartnerTeamsCallHoldPolicy" -Description "country music" -AudioFileID "c65233-ac2a27-98701b-123ccc" 

```

File: skype/skype-ps/skype/grant-CsTeamsCallHoldPolicy.md
Comment (get-itips) [inline skype/skype-ps/skype/grant-CsTeamsCallHoldPolicy.md:96]: Seems that including `-Global` makes the selected audio the new Global

image
image

Can you kindly state this in the description? Thank you

**Review by get-itips — CHANGES_REQUESTED** [review]
Please review my comments and also replace all occurrences of "Teams compliance recording policies" to "Teams call hold policies"

**Review by tseward — APPROVED** [review]

File: skype/skype-ps/skype/Remove-CsTeamsCallHoldPolicy.md
Comment (dariomws) [inline skype/skype-ps/skype/Remove-CsTeamsCallHoldPolicy.md:35]:
```suggestion
### Example 1

```

File: skype/skype-ps/skype/Remove-CsTeamsCallHoldPolicy.md
Comment (dariomws) [inline skype/skype-ps/skype/Remove-CsTeamsCallHoldPolicy.md:140]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: skype/skype-ps/skype/Set-CsTeamsCallHoldPolicy.md
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsTeamsCallHoldPolicy.md:199]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: skype/skype-ps/skype/grant-CsTeamsCallHoldPolicy.md
Comment (dariomws) [inline skype/skype-ps/skype/grant-CsTeamsCallHoldPolicy.md:208]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: skype/skype-ps/skype/Set-CsTeamsCallHoldPolicy.md
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsTeamsCallHoldPolicy.md:59]:
```suggestion

```

Comment (dariomws) [reply skype/skype-ps/skype/Set-CsTeamsCallHoldPolicy.md:59]: I think this sentence was copied and pasted from example 1, but it's not applicable here.

**Review by dariomws — CHANGES_REQUESTED** [review]
Agree with @get-itips and also added some more suggestions. Thanks!

**Comment by yogkumgit** [comment]
@joelhmarquez Please review and implement the suggested changes

**Comment by yogkumgit** [comment]
@joelhmarquez Gentle reminder

**Comment by joelhmarquez** [comment]
Thanks for all the suggestions! I will implement them and push an update shortly.

**Comment by yogkumgit** [comment]
@joelhmarquez

File: skype/skype-ps/skype/grant-CsTeamsCallHoldPolicy.md
Comment (joelhmarquez) [reply skype/skype-ps/skype/grant-CsTeamsCallHoldPolicy.md:103]: Added another example showing this

**Comment by joelhmarquez** [comment]
Thanks again for all the suggestions, I have gone ahead and resolved the remaining ones.

I was asked if we can hold off on merging this until 6/15 to wait for another set of cmdlets to become available at which point the feature in which the call hold policy cmdlets will be used will be announced.

I will post here again once the changes we are waiting on are ready and then we are good to publish these.

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Comment by Kateyanne** [comment]
@JohanFreelancer9 please copyedit this PR. Thank you!

**Comment by joelhmarquez** [comment]
@Kateyanne can we actually hold off on merging for a couple more days. I was told to hold off on making these docs public until another set of cmdlets are ready (should be ready this week) since they are related and are to be used together for the custom music on hold scenarios and thus we want to avoid confusion by releasing these documentation before making the cmdlets public. 

If we do need to merge it now, we can do so but we should add a preview label so that people are aware that it is not publicly available yet. 

If we are okay holding off a little bit, I can update here as soon as I get confirmation that the other set of cmdlets are ready.

**Comment by Kateyanne** [comment]
@joelhmarquez I have added the 'do not merge' label. When you feel this is ready to be merged, let me know.

File: skype/skype-ps/skype/Grant-CsTeamsCallHoldPolicy.md
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsCallHoldPolicy.md:155]:
```suggestion
The PolicyName is simply the policy Identity without the policy scope, i.e. the "Tag:" prefix.

```

File: skype/skype-ps/skype/New-CsTeamsCallHoldPolicy.md
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsCallHoldPolicy.md:41]:
```suggestion
This policy is assigned the audio file id: c65233-ac2a27-98701b-123ccc, which is the ID referencing an audio file that was uploaded using the Import-CsOnlineAudioFile cmdlet.

```

File: skype/skype-ps/skype/New-CsTeamsCallHoldPolicy.md
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsCallHoldPolicy.md:48]:
```suggestion
A string representing the ID referencing an audio file uploaded via the Import-CsOnlineAudioFile cmdlet.

```

File: skype/skype-ps/skype/Set-CsTeamsCallHoldPolicy.md
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsCallHoldPolicy.md:47]:
```suggestion
This policy is re-assigned the audio file ID to be used to: c65233-ac2a27-98701b-123ccc, which is the ID referencing an audio file that was uploaded using the Import-CsOnlineAudioFile cmdlet.

```

File: skype/skype-ps/skype/Set-CsTeamsCallHoldPolicy.md
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsCallHoldPolicy.md:63]:
```suggestion
A string representing the ID referencing an audio file uploaded via the Import-CsOnlineAudioFile cmdlet.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few additional minor suggestions.

**Comment by yogkumgit** [comment]
@joelhmarquez Please implement the suggested changes. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by joelhmarquez** [comment]
Hi @Kateyanne, I know it's been a while but we've now got approval to go ahead and merge this. Can you help me get it merged? Thank you!

**Comment by get-itips** [comment]
> Hi @Kateyanne, I know it's been a while but we've now got approval to go ahead and merge this. Can you help me get it merged? Thank you!

@tiburd

**Comment by tiburd** [comment]
@get-itips - I'll merge it.

**Comment by traya1** [comment]
Hi Team, just a reminder that we do not to merge PRs with outstanding warnings. I've resolved the warnings created in this PR.

Thanks!

**Pull request merged**
